mnist/README.md

```diff
@@ -6,165 +6,96 @@ This example guides you through the process of taking an example model, modifyin
-
-### Deploying Tensorflow Operator and Argo.
 
-We are using the Tensorflow operator to automate the deployment of our distributed model training, and Argo to create the overall training pipeline. The easiest way to install these components on your Kubernetes cluster is by using Kubeflow's ksonnet prototypes.
+## 5.Download argo cli
 
 ```
-NAMESPACE=tfworkflow
-APP_NAME=my-kubeflow
-ks init ${APP_NAME}
-cd ${APP_NAME}
-
-ks registry add kubeflow github.com/kubeflow/kubeflow/tree/v0.2.4/kubeflow
-
-ks pkg install kubeflow/core@v0.2.4
-ks pkg install kubeflow/argo
-
-# Deploy TF Operator and Argo
-kubectl create namespace ${NAMESPACE}
-ks generate core kubeflow-core --name=kubeflow-core --namespace=${NAMESPACE}
-ks generate argo kubeflow-argo --name=kubeflow-argo --namespace=${NAMESPACE}
-
-ks apply default -c kubeflow-core
-ks apply default -c kubeflow-argo
-
-# Switch context for the rest of the example
-kubectl config set-context $(kubectl config current-context) --namespace=${NAMESPACE}
-cd -
-
-# Create a user for our workflow
-kubectl apply -f tf-user.yaml
+curl -sSL -o /usr/local/bin/argo https://github.com/argoproj/argo/releases/download/v2.2.1/argo-linux-amd64
+chmod +x /usr/local/bin/argo
 ```
 
-You can check to make sure the components have deployed:
+## 6.Create tf service account
 
-```
-$ kubectl get pods -l name=tf-job-operator
-NAME                              READY     STATUS    RESTARTS   AGE
-tf-job-operator-78757955b-2glvj   1/1       Running   0          1m
+Due to lack of permissions in default serviceaccount, so we created a new one with the required permissions.
 
-$ kubectl get pods -l app=workflow-controller
-NAME                                   READY     STATUS    RESTARTS   AGE
-workflow-controller-7d8f4bc5df-4zltg   1/1       Running   0          1m
+We will use `mnist` namespace for mnist model.
 
-$ kubectl get crd
-NAME                    AGE
-tfjobs.kubeflow.org     1m
-workflows.argoproj.io   1m
+```
+export NAMESPACE=mnist
 
-$ argo list
-NAME   STATUS   AGE   DURATION
+kubectl create ns ${NAMESPACE}
+kubectl -n ${NAMESPACE} apply -f tf-user.yaml
 ```
 
-### Creating secrets for our workflow and setting S3 variables.
+## 7.Create secret for workflow
 
-For fetching and uploading data, our workflow requires S3 credentials and variables. These credentials will be provided as kubernetes secrets, and the variables will be passed into the workflow. Modify the below values to suit your environment.
+NOTE: replace `192.168.100.155` with your own Minio server address that setup in step 4.
 
 ```
-export S3_ENDPOINT=s3.us-west-2.amazonaws.com  #replace with your s3 endpoint in a host:port format, e.g. minio:9000
-export AWS_ENDPOINT_URL=https://${S3_ENDPOINT} #use http instead of https for default minio installs
-export AWS_ACCESS_KEY_ID=xxxxx
-export AWS_SECRET_ACCESS_KEY=xxxxx
+export S3_ENDPOINT=192.168.100.155:9000
```

Comment:
This makes a lot of assumptions about the network configuration.

Reply:
in line 78, there is a description about this.